docs/reference/performance.rst

```diff
@@ -0,0 +1,277 @@
+.. meta::
+  :description: This chapter describes a set of best practices designed to help
+   developers optimize the performance of HIP-capable GPU architectures.
+  :keywords: AMD, ROCm, HIP, CUDA, performance, guidelines
+
+*******************************************************************************
+Performance Guidelines
+*******************************************************************************
+
+The AMD HIP Performance Guidelines are a set of best practices designed to help
+developers optimize the performance of AMD GPUs. They cover established
+parallelization and optimization techniques, coding metaphors, and idioms that
+can greatly simplify programming for HIP-capable GPU architectures.
+
+By following four main cornerstones, we can exploit the performance 
+optimization potential of HIP.
+
+- parallel execution
+- memory usage optimization
+- optimization for maximum instruction
```

Comment:
This is about throughput effectively. Will be better
```suggestion
- optimization for maximum throughput
```

Reply:
Thanks! Fixed.